### PR TITLE
Add placeholder on album without cover

### PR DESCRIPTION
**Pull request by GoOz**
And alos fixed height of album cover so it doesn't "bounce" when loaded

### CODE REVIEW

**Comment by GoOz** [comment]
[(0.5) [PHO] Améliorer le placeholder de la cover d'un album vide](https://trello.com/c/dFVIryW3/304-05-pho-am%C3%A9liorer-le-placeholder-de-la-cover-dun-album-vide)

File: src/components/ImageLoader.jsx [hunk 1]
```diff
@@ -9,12 +9,10 @@ const getPhotoLink = async (photoId) => {
 
 export default class ImageLoader extends Component {
   state = {
-    loading: true,
```
Comment (GoOz) [inline src/components/ImageLoader.jsx]: Not sure about all the loading var though, but i can't any other use anywhere.

Comment (enguerran) [inline src/components/ImageLoader.jsx]: 👍

**Comment by enguerran** [comment]
🔉 :great:

**Pull request merged**